<?xml version="1.0" encoding="utf-8"?>
<!-- This URDF was automatically created by SolidWorks to URDF Exporter! Originally created by Stephen Brawner (brawner@gmail.com) 
     Commit Version: 1.6.0-4-g7f85cfe  Build Version: 1.6.7995.38578
     For more information, please see http://wiki.ros.org/sw_urdf_exporter -->
<robot
  name="quad_robo">
  <link
    name="body">
    <inertial>
      <origin
        xyz="0.131306204381629 0.545054702211338 0.879506462369344"
        rpy="0 0 0" />
      <mass
        value="79.5537197619098" />
      <inertia
        ixx="1.64089584946241"
        ixy="-5.94676449636783E-07"
        ixz="-3.23528257797902E-06"
        iyy="7.3876567503266"
        iyz="1.08488701873935E-07"
        izz="6.25234462192458" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://quad_robo/meshes/body.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0 0.752941176470588 0 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://quad_robo/meshes/body.STL" />
      </geometry>
    </collision>
  </link>
  <link
    name="camera">
    <inertial>
      <origin
        xyz="-5.26641064091393E-09 0.0232978247256392 5.17607523509156E-08"
        rpy="0 0 0" />
      <mass
        value="0.0738276082029152" />
      <inertia
        ixx="2.02746400750097E-05"
        ixy="-5.87121077446652E-12"
        ixz="5.41842911576072E-12"
        iyy="2.05931319726044E-05"
        iyz="5.76026306050092E-11"
        izz="2.02745272851038E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://quad_robo/meshes/camera.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0 0 0 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://quad_robo/meshes/camera.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="camera_joint"
    type="fixed">
    <origin
      xyz="0.53131 -0.88451 0.64005"
      rpy="1.5708 0 0" />
    <parent
      link="body" />
    <child
      link="camera" />
    <axis
      xyz="0 1 0" />
  </joint>
  <link
    name="back_right_thigh">
    <inertial>
      <origin
        xyz="0.062909843287888 -5.55111512312578E-16 0.0554258383771329"
        rpy="0 0 0" />
      <mass
        value="2.77020343496813" />
      <inertia
        ixx="0.0122323817817133"
        ixy="1.41268154943072E-17"
        ixz="-0.00138250516290713"
        iyy="0.0121646709526814"
        iyz="5.36646194062435E-17"
        izz="0.00671928758663995" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://quad_robo/meshes/back_right_thigh.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.250980392156863 0.250980392156863 0.250980392156863 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://quad_robo/meshes/back_right_thigh.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="back_right_thigh_joint"
    type="revolute">
    <origin
      xyz="-0.11864 -1.0595 0.54505"
      rpy="-1.5785 0 3.1416" />
    <parent
      link="body" />
    <child
      link="back_right_thigh" />
    <axis
      xyz="-1 0 0" />
    <limit
      lower="-3.14"
      upper="3.14"
      effort="0"
      velocity="0" />
  </joint>
  <link
    name="back_right_leg">
    <inertial>
      <origin
        xyz="0.0730352545446933 -0.237417764752276 6.5188398845617E-09"
        rpy="0 0 0" />
      <mass
        value="6.93816869174773" />
      <inertia
        ixx="0.144761661754863"
        ixy="-0.00478965570518421"
        ixz="-3.98269493506236E-08"
        iyy="0.0173922214758361"
        iyz="6.48408997339703E-09"
        izz="0.145237872165608" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://quad_robo/meshes/back_right_leg.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.501960784313725 1 0.501960784313725 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://quad_robo/meshes/back_right_leg.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="back_right_leg_joint"
    type="revolute">
    <origin
      xyz="0.07005 0 0.070008"
      rpy="0.012688 -1.5708 0" />
    <parent
      link="back_right_thigh" />
    <child
      link="back_right_leg" />
    <axis
      xyz="1 0 0" />
    <limit
      lower="-3.14"
      upper="3.14"
      effort="0"
      velocity="0" />
  </joint>
  <link
    name="back_right_shin">
    <inertial>
      <origin
        xyz="0.0699916518608028 0.206758869782017 -2.45496274753343E-07"
        rpy="0 0 0" />
      <mass
        value="2.18909073352721" />
      <inertia
        ixx="0.0495544592119496"
        ixy="-4.02316241301775E-09"
        ixz="-1.2891756100279E-08"
        iyy="0.00191161408925373"
        iyz="5.10972999990865E-09"
        izz="0.0499442056233554" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://quad_robo/meshes/back_right_shin.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0 0 0 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://quad_robo/meshes/back_right_shin.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="back_right_shin_joint"
    type="revolute">
    <origin
      xyz="0 -0.5 0"
      rpy="3.1324 0 0" />
    <parent
      link="back_right_leg" />
    <child
      link="back_right_shin" />
    <axis
      xyz="-1 0 0" />
    <limit
      lower="-3.14"
      upper="3.14"
      effort="0"
      velocity="0" />
  </joint>
  <link
    name="back_left_thigh">
    <inertial>
      <origin
        xyz="0.062909843287888 -3.33066907387547E-16 0.0554258383771333"
        rpy="0 0 0" />
      <mass
        value="2.77020343496813" />
      <inertia
        ixx="0.0122323817817133"
        ixy="6.46455545344482E-18"
        ixz="-0.00138250516290713"
        iyy="0.0121646709526814"
        iyz="2.54787510534094E-17"
        izz="0.00671928758663995" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://quad_robo/meshes/back_left_thigh.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.250980392156863 0.250980392156863 0.250980392156863 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://quad_robo/meshes/back_left_thigh.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="back_left_thigh_joint"
    type="revolute">
    <origin
      xyz="-0.11864 -0.69951 0.54505"
      rpy="1.5468 0 3.1416" />
    <parent
      link="body" />
    <child
      link="back_left_thigh" />
    <axis
      xyz="-1 0 0" />
    <limit
      lower="-3.14"
      upper="3.14"
      effort="0"
      velocity="0" />
  </joint>
  <link
    name="back_left_leg">
    <inertial>
      <origin
        xyz="0.072963236277204 -0.237417763004437 -5.14056230915827E-09"
        rpy="0 0 0" />
      <mass
        value="6.93816842607885" />
      <inertia
        ixx="0.144761657967399"
        ixy="-0.00478964566005541"
        ixz="-4.03728476960052E-08"
        iyy="0.017392218412496"
        iyz="9.8938174064348E-09"
        izz="0.145237864969177" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://quad_robo/meshes/back_left_leg.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.501960784313725 1 0.501960784313725 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://quad_robo/meshes/back_left_leg.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="back_left_leg_joint"
    type="revolute">
    <origin
      xyz="0.07005 0 0.07008"
      rpy="1.5708 -1.5708 -1.5553" />
    <parent
      link="back_left_thigh" />
    <child
      link="back_left_leg" />
    <axis
      xyz="1 0 0" />
    <limit
      lower="-3.14"
      upper="3.14"
      effort="0"
      velocity="0" />
  </joint>
  <link
    name="back_left_shin">
    <inertial>
      <origin
        xyz="0.069919797930706 0.206758606104935 2.62203067119549E-07"
        rpy="0 0 0" />
      <mass
        value="2.18909502281746" />
      <inertia
        ixx="0.0495545339536704"
        ixy="-9.70636337147349E-11"
        ixz="-8.11442350270843E-09"
        iyy="0.00191161861589561"
        iyz="-1.96303128510681E-08"
        izz="0.0499442789876987" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://quad_robo/meshes/back_left_shin.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0 0 0 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://quad_robo/meshes/back_left_shin.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="back_left_shin_joint"
    type="revolute">
    <origin
      xyz="0 -0.5 0"
      rpy="-3.1314 0 0" />
    <parent
      link="back_left_leg" />
    <child
      link="back_left_shin" />
    <axis
      xyz="-1 0 0" />
    <limit
      lower="-3.14"
      upper="3.14"
      effort="0"
      velocity="0" />
  </joint>
  <link
    name="front_left_thigh">
    <inertial>
      <origin
        xyz="0.062909843287888 -1.11022302462516E-16 0.0554258383771333"
        rpy="0 0 0" />
      <mass
        value="2.77020343496812" />
      <inertia
        ixx="0.0122323817817133"
        ixy="2.80537312130624E-18"
        ixz="-0.00138250516290713"
        iyy="0.0121646709526814"
        iyz="1.53956708492942E-17"
        izz="0.00671928758663994" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://quad_robo/meshes/front_left_thigh.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.250980392156863 0.250980392156863 0.250980392156863 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://quad_robo/meshes/front_left_thigh.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="front_left_thigh_joint"
    type="revolute">
    <origin
      xyz="0.38126 -0.69951 0.54505"
      rpy="-1.541 0 0" />
    <parent
      link="body" />
    <child
      link="front_left_thigh" />
    <axis
      xyz="-1 0 0" />
    <limit
      lower="-3.14"
      upper="3.14"
      effort="0"
      velocity="0" />
  </joint>
  <link
    name="front_left_leg">
    <inertial>
      <origin
        xyz="0.0729197874329797 -0.23741776169486 3.33459237999989E-09"
        rpy="0 0 0" />
      <mass
        value="6.93816836078529" />
      <inertia
        ixx="0.144761656790624"
        ixy="-0.00478964515103163"
        ixz="-3.99813356402284E-08"
        iyy="0.0173922180334012"
        iyz="7.55877140204142E-09"
        izz="0.145237863562429" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://quad_robo/meshes/front_left_leg.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.501960784313725 1 0.501960784313725 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://quad_robo/meshes/front_left_leg.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="front_left_leg_joint"
    type="revolute">
    <origin
      xyz="0.07005 0 0.070124"
      rpy="-1.5708 -1.5708 -1.5798" />
    <parent
      link="front_left_thigh" />
    <child
      link="front_left_leg" />
    <axis
      xyz="1 0 0" />
    <limit
      lower="-3.14"
      upper="3.14"
      effort="0"
      velocity="0" />
  </joint>
  <link
    name="front_left_shin">
    <inertial>
      <origin
        xyz="0.0698764395589798 0.206758862215771 1.34053533940026E-07"
        rpy="0 0 0" />
      <mass
        value="2.18909066860029" />
      <inertia
        ixx="0.0495544565143451"
        ixy="2.7795255573489E-09"
        ixz="-1.12016659458804E-08"
        iyy="0.00191161192130292"
        iyz="-2.86451210553766E-08"
        izz="0.0499442043702863" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://quad_robo/meshes/front_left_shin.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0 0 0 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://quad_robo/meshes/front_left_shin.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="front_left_shin_joint"
    type="revolute">
    <origin
      xyz="0 -0.5 0"
      rpy="-3.1181 0 0" />
    <parent
      link="front_left_leg" />
    <child
      link="front_left_shin" />
    <axis
      xyz="-1 0 0" />
    <limit
      lower="-3.14"
      upper="3.14"
      effort="0"
      velocity="0" />
  </joint>
  <link
    name="front_right_thigh">
    <inertial>
      <origin
        xyz="0.062909843287888 1.11022302462516E-16 0.0554258383771331"
        rpy="0 0 0" />
      <mass
        value="2.77020343496813" />
      <inertia
        ixx="0.0122323817817133"
        ixy="-2.30223555063719E-18"
        ixz="-0.00138250516290713"
        iyy="0.0121646709526814"
        iyz="-8.98532550447362E-18"
        izz="0.00671928758663995" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://quad_robo/meshes/front_right_thigh.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.250980392156863 0.250980392156863 0.250980392156863 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://quad_robo/meshes/front_right_thigh.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="front_right_thigh_joint"
    type="revolute">
    <origin
      xyz="0.38126 -1.0595 0.54505"
      rpy="1.5765 0 0" />
    <parent
      link="body" />
    <child
      link="front_right_thigh" />
    <axis
      xyz="-1 0 0" />
    <limit
      lower="-3.14"
      upper="3.14"
      effort="0"
      velocity="0" />
  </joint>
  <link
    name="front_right_leg">
    <inertial>
      <origin
        xyz="0.07303901744713 -0.23741776527427 -3.69308134073876E-09"
        rpy="0 0 0" />
      <mass
        value="6.93816871372888" />
      <inertia
        ixx="0.144761662160028"
        ixy="-0.00478965550109165"
        ixz="-4.02638939699464E-08"
        iyy="0.0173922216019095"
        iyz="9.32153008030453E-09"
        izz="0.145237872671702" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://quad_robo/meshes/front_right_leg.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.501960784313725 1 0.501960784313725 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://quad_robo/meshes/front_right_leg.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="front_right_leg_joint"
    type="revolute">
    <origin
      xyz="0.07005 0 0.070005"
      rpy="1.5836 -1.5708 -1.5669" />
    <parent
      link="front_right_thigh" />
    <child
      link="front_right_leg" />
    <axis
      xyz="1 0 0" />
    <limit
      lower="-3.14"
      upper="3.14"
      effort="0"
      velocity="0" />
  </joint>
  <link
    name="front_right_shin">
    <inertial>
      <origin
        xyz="0.0699954264525831 0.20675886909659 -1.74080901305107E-07"
        rpy="0 0 0" />
      <mass
        value="2.18909072827596" />
      <inertia
        ixx="0.0495544584922607"
        ixy="-1.29555843858146E-09"
        ixz="-1.28395081007654E-08"
        iyy="0.00191161363476337"
        iyz="7.95536208459755E-10"
        izz="0.0499442059837185" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://quad_robo/meshes/front_right_shin.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0 0 0 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://quad_robo/meshes/front_right_shin.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="front_right_shin_joint"
    type="revolute">
    <origin
      xyz="0 -0.5 0"
      rpy="3.1105 0 0" />
    <parent
      link="front_right_leg" />
    <child
      link="front_right_shin" />
    <axis
      xyz="-1 0 0" />
    <limit
      lower="-3.14"
      upper="3.14"
      effort="0"
      velocity="0" />
  </joint>
</robot>